src/Sarif/Schemata/sarif-schema.json

```diff
@@ -715,6 +710,16 @@
           "type": "integer"
         },
 
+        "ruleConfiguration": {
```

Comment:
>ruleConfiguration [](start = 9, length = 17)

Ah, yes, I remember discussing how this is really a property of the invocation. Nice.